File: cloud-collect-core/src/main/java/com/windea/demo/cloudcollect/core/exception/NotFoundException.java
package com.windea.demo.cloudcollect.core.exception;

/**
 * 页面未找到的异常。
 */
public class NotFoundException extends RuntimeException {
	private static final long serialVersionUID = 3747208892156399108L;

	private static final String message = "404 页面未找到！";

	public NotFoundException() {
		super(message);
	}

	public NotFoundException(Throwable throwable) {
		super(message, throwable);
	}
}

File: cloud-collect-core/src/main/kotlin/com/windea/demo/cloudcollect/core/exception/NotFoundException.kt
package com.windea.demo.cloudcollect.core.exception

import com.windea.demo.cloudcollect.core.domain.enums.*

/**页面未找到时的异常。 */
class NotFoundException : RuntimeException {
	constructor() : super(ResponseResult.NOT_FOUND.message)
	
	constructor(throwable: Throwable) : super(ResponseResult.NOT_FOUND.message, throwable)
}
